src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,85 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private Node<K, V>[] nodeArray = new Node[DEFAULT_CAPACITY];
```

Comment:
```suggestion
    private Node<K, V>[] buckets = new Node[DEFAULT_CAPACITY];
```
name can be a bit misleading